operational_analysis/methods/electrical_losses.py

```diff
@@ -73,17 +64,27 @@ def __init__(self, plant, UQ = False, num_sim = 20000, uncertainty_meter=0.005,
         self._hours_per_day= 24 # Hours per day converter
     
     @logged_method_call
-    def run(self):
+    def run(self, uncertainty_meter=0.005, uncertainty_scada=0.005,
+                 uncertainty_correction_thresh=(0.9,0.995)):
```

Comment:
How is the correction threshold specified if the user wants to use UQ = False? In the original version, correction_thresh was an input that defaulted to 0.95. It appears this is hardcoded now on line 127.

Reply:
Thanks for catching this. I have now changed this.